File: Level 1/LoginValidation.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {

    string username;
    string password;

    cout << "Enter username: ";
    cin >> username;

    cout << "Enter password: ";
    cin >> password;

    if(username == "adminKE" && password == "254Secure") {
        cout << "Access Granted";
    } else {
        cout << "Invalid Credentials";
    }

    return 0;
}

File: Level 1/LoginValidation.java
import java.util.Scanner;

public class LoginValidation {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter username: ");
        String username = scanner.nextLine();

        System.out.print("Enter password: ");
        String password = scanner.nextLine();

        if(username.equals("adminKE") && password.equals("254Secure")){
            System.out.println("Access Granted");
        } else {
            System.out.println("Invalid Credentials");
        }

            scanner.close();
    }
}